modules/identity-provider-add.adoc

```diff
@@ -37,15 +48,41 @@ $ oc apply -f </path/to/CR>
 If a CR does not exist, `oc apply` creates a new CR and might trigger the following warning: `Warning: oc apply should be used on resources created by either oc create --save-config or oc apply`. In this case you can safely ignore this warning.
 ====
 
+ifndef::no-username-password-login[]
 . Log in to the cluster as a user from your identity provider, entering the
 password when prompted.
 +
 ----
 $ oc login -u <username>
 ----
+endif::no-username-password-login[]
+
+ifdef::no-username-password-login[]
+. Log in to the cluster, passing in the token to authenticate.
++
+----
+$ oc login --token=<token>
+----
++
+[NOTE]
+====
+This identity provider does not support logging in with a username and password.
```

Comment:
s/username/user name

Reply:
I hate that guidance but I'll change it :smile: